src/plugins/comments/comments.js

```diff
@@ -734,6 +737,10 @@ class Comments extends BasePlugin {
    * @param {object} defaultOptions The menu options.
    */
   addToContextMenu(defaultOptions) {
+    const isThereAnyCellRendered = () => {
+      return this.hot.view.countRenderableRows() > 0 && this.hot.view.countRenderableColumns() > 0;
```

Comment:
Should we use methods? WDYT?

Reply:
Good point, I use IndexMapper methods https://github.com/handsontable/handsontable/pull/7027/commits/8fadcf39f47c345a02a162fcfd5ff4aaf4fc6ff5.

Reply:
Although, they don't have information about `maxRows` and `maxCols`. I'm not sure if this is the best option 🤔

Reply:
For comments menu items it is not important though.